Jetchat/app/src/main/java/com/example/compose/jetchat/components/JetchatAppBar.kt

```diff
@@ -43,21 +44,23 @@ fun JetchatAppBar(
     title: @Composable () -> Unit,
     actions: @Composable RowScope.() -> Unit = {}
 ) {
-    CenterAlignedTopAppBar(
-        modifier = modifier,
-        actions = actions,
-        title = title,
-        scrollBehavior = scrollBehavior,
-        navigationIcon = {
-            JetchatIcon(
-                contentDescription = stringResource(id = R.string.navigation_drawer_open),
-                modifier = Modifier
-                    .size(64.dp)
-                    .clickable(onClick = onNavIconPressed)
-                    .padding(16.dp)
-            )
-        }
-    )
+    Box {
+        CenterAlignedTopAppBar(
+            modifier = modifier,
```

Comment:
Should this modifier be on the parent Box now?

Reply:
Yes this solution comes across as a hack. @JolandaVerhoef will try adding a `Scaffold` to the app which should be the correct way to solve the overlap issue.